control.c

```diff
@@ -16,8 +16,10 @@ static volatile bool collision_detected_signal;
 static volatile bool motor_control_enabled_signal;
 static volatile bool side_sensors_control_enabled;
 static volatile bool front_sensors_control_enabled;
+static volatile bool diagonal_enabled;
 static volatile float side_sensors_integral;
 static volatile float front_sensors_integral;
+static volatile float diagonal_integral;
```

Comment:
I would use a more consistent naming, like `diagonal_sensors_control_enabled` and `diagonal_sesors_integral`.

Reply:
90e89df9ff933